lib/plugin.client.js

```diff
@@ -10,7 +10,13 @@ import { Integrations as TracingIntegrations } from '@sentry/tracing'
 export default function (ctx, inject) {
   <% if (options.initialize) { %>
   /* eslint-disable object-curly-spacing, quote-props, quotes, key-spacing, comma-spacing */
-  const config = <%= serialize(options.config) %>
+  const config = <%= JSON.stringify(options.config) %>
+
+  const publicRuntimeConfigKey = <%= serialize(options.publicRuntimeConfigKey) %>
+  if (ctx.$config && publicRuntimeConfigKey) {
+    Object.assign(config, ctx.$config[publicRuntimeConfigKey])
```

Comment:
`Object.assign` is a very poor way of merging options. Should use `lodash.merge` instead.

(Currently this module uses `deepmerge` for merging options but we shouldn't use that one on the client so we can replace `deepmerge` with `lodash.merge` altogether.)

Reply:
Ok, I'll do the replacement.